src/site/layouts/health_care_region_detail_page.drupal.liquid

```diff
@@ -19,7 +19,8 @@
           </div>
 
           {% assign isContactPage = entityUrl.path | isPage: "contact-us" %}
-          {% if isContactPage %}
+          {% assign isProgramsChildPage = entityUrl | isChildPageOf: "Programs" %}
```

Comment:
Is this synonymous with the following JS?

```js
const isProgramsChildPage = isChildPageOf('Programs', entityUrl);
```

So `isProgramsChildPage` is either `true` or `false`, and never `entityUrl`?

Reply:
yes